service/resource/endpoint/create.go

```diff
@@ -5,73 +5,34 @@ import (
 
 	"github.com/giantswarm/microerror"
 	"k8s.io/apimachinery/pkg/api/errors"
-	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	apiv1 "k8s.io/client-go/pkg/api/v1"
 )
 
 func (r *Resource) ApplyCreateChange(ctx context.Context, obj, createState interface{}) error {
-	endpointToCreate, err := toEndpoint(createState)
+	k8sEndpoint, err := toK8sEndpoint(createState)
 	if err != nil {
 		return microerror.Mask(err)
 	}
 
-	if len(endpointToCreate.IPs) == 0 {
-		// Nothing to do.
+	_, err = r.k8sClient.CoreV1().Endpoints(k8sEndpoint.Namespace).Create(k8sEndpoint)
```

Comment:
In the operators is usually a check if the state is `nil` to make sure we have to do work or not. Then we also have logging.

Reply:
Added the check, logging will be a separate PR.